Formula/c/cody.rb

```diff
@@ -0,0 +1,26 @@
+class Cody < Formula
+  desc "Sourcegraph Cody CLI"
+  homepage "https://sourcegraph.com"
+  url "https://registry.npmjs.org/@sourcegraph/cody/-/cody-5.5.15.tgz"
+  sha256 "2f25357f26e7710b71e4abead3ad9ee3fc5817fbbd6a78cd6726556982bc935a"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    ENV["NPM_CONFIG_PREFIX"] = libexec.to_s
+    system "npm", "install", *std_npm_args, "@sourcegraph/cody"
```

Comment:
This will just install any version right? It won't install from the source that's downloaded.

Reply:
Thanks @SMillerDev  for the review; version now matches source code for better control.